pkg/tfbridge/schema.go

```diff
@@ -1305,7 +1305,7 @@ func MakeTerraformState(
 		// If there was no metadata in the inputs and this resource has a non-zero
 		// schema version, return a meta bag with the current schema version. This
 		// helps avoid migration issues.
-		meta = map[string]interface{}{"schema_version": strconv.Itoa(res.TF.SchemaVersion())}
+		meta = map[string]interface{}{"schema_version": "0"}
```

Comment:
It sounds like a directionally correct but dangerous change because the previous behavior might have been intentional. We'll now be opening the door to these migration issues that didn't use to be present before, since migrators will now start running, and panicking because we do not support RawState parsing properly.

Certainly would like to see this flagged so provider can opt in or out. Consider also a gradual rollout strategy.

Reply:
Yeah, absolutely. RawState is fixed under PRC AFAIK, should we only enable this under PRC?

Also it seems like if the migration doesn't run we still are expecting bad things/ panics, so I'm not sure this really makes it worse.

Reply:
Yup, I see what you mean. Should we add a provider flag here or a per-resource flag? Also are we happy to ship this by default and work around any issues which might arise later?

Reply:
I think start with a provider flag that the user can override in case of an emergency via env var.

Reply:
This is a tough call, I think I'm not comfortable with shipping by default and would like a small ladder like enable in GCP first, AWS next.